src/components/timeline/Track.css

```diff
@@ -165,8 +167,9 @@
 }
 
 .timelineTrackLocalLabel {
-  /* We want the width to look like it's 150px, but need to substract the 3px padding/margin and the 1px border */
-  width: calc(135px - var(--selected-left-border-width) - 1px);
+  /* We want the width to look like it's 150px, but need to substract the
+   * margin and the 1px border */
```

Comment:
I think the comment is incorrect, it was copied from `timelineTrackLabel`.

It might be worth using variables for the global trrack's width (150px) and the local track's margin (15px), and still use a `calc` here for documentation purpose?